pkg/ddevapp/apptypes.go

```diff
@@ -133,6 +133,10 @@ func (app *DdevApp) CreateSettingsFile() (string, error) {
 		if err != nil {
 			util.Warning("Unable to create settings file: %v", err)
 		}
+		if err := CreateGitIgnore(filepath.Dir(app.SiteSettingsPath), filepath.Base(app.SiteLocalSettingsPath), "ddev_drush_settings.php"); err != nil {
```

Comment:
This may be my CMS ignorance showing, but isn't this very drupal-centric? Do TYPO3 and Wordpress projects have a `ddev_drush_settings.php` file?

I suppose there's no harm in ignoring something that may not exist, but I think this block could be flagged for a CMS-specific refactor.

Reply:
My assumption is that it's ok to ignore something that doesn't exist to make the code simpler. Drush is for Drupal only, but I guess it shouldn't be offensive to people to find a non-their-CMS file gitignored.

This is actually *undoing* the previous CMS-specific approach, which involved 4x duplicated code.

Reply:
Gotcha, I see that now. That's fine with me.